src/core/inline-idl-parser.js

```diff
@@ -183,7 +183,7 @@ function renderAttribute(details) {
 function renderMethod(details) {
   const { args, identifier, type, parent, renderParent } = details;
   const { identifier: linkFor } = parent || {};
-  const argsText = args.map(arg => `<var>${arg}</var>`).join(", ");
+  const argsText = args.map(arg => hyperHTML `<var>${arg}</var>`).join(", ");
```

Comment:
`.join(", ")` won't work anymore because hyperHTML will return an HTMLElement here. We need a clever way to get an array with `", "` between each item, and then we can pass it instead of `[argText]` at line 193.